Week2/homework/maartjes_work.js

```diff
@@ -45,3 +45,34 @@ const tuesday = [
 const tasks = monday.concat(tuesday);
 
 // Add your code here
+
+
+//Step-1: Map the tasks to durations in hours.
+const workHour = tasks.map(myElement =>
+  (Math.round(100 * (myElement.duration / 60))) / 100);
```

Comment:
same result.
```javascript
// Yours
(Math.round(100 * (myElement.duration / 60))) / 100
// Mine 😃
Math.floor(myElement.duration / .6) / 100
```
.6 is used to show duration in hour.
100 is used to round with two decimals such as 0.33 or 1.23

Reply:
the both of them can be used, right? 
Math.round(myElement.duration / .6) / 100
Math.floor(myElement.duration / .6) / 100
They look like each other :)

Reply:
Yes exactly you can use any of them both, the only deffirent is floor always giving you the nearset lower value, nor round giving you the nearest value of course depeniding on the case you have. 👍